lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -347,10 +351,10 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
         return inner
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 3, 4, 21),
-        (ModuleStoreEnum.Type.mongo, 20, 4, 21),
-        (ModuleStoreEnum.Type.split, 3, 13, 21),
-        (ModuleStoreEnum.Type.split, 20, 13, 21),
+        (ModuleStoreEnum.Type.mongo, 3, 4, 22),
```

Comment:
The problem is that if we don't address it here then we are contributing to the query creep on the platform. Adding in the cache you discussed with Ben will help for non-team topics, so maybe that's enough. In general, I don't know the best approach to keeping the platform query load steady while also adding new capabilities.